etc/linters.json

```diff
@@ -2,27 +2,152 @@
     {
         "name": "ruff",
         "args": "check --config /etc/opt/goat/pyproject.toml -v",
-        "filetype": ".py",
-        "installer": "pip"
+        "filetype": [
+            ".py"
+        ],
+        "installer": "pip",
+        "env": "RUFF_CONFIG"
+    },
+    {
+        "name": "hadolint",
+        "args": "-V -c /etc/opt/goat/.hadolint.yml",
+        "filetype": [
+            "Dockerfile"
+        ],
+        "installer": "docker",
+        "env": "HADOLINT_CONFIG"
+    },
+    {
+        "name": "kubeconform",
+        "args": "-verbose -strict",
+        "filetype": [
+            ".yml",
+            ".yaml"
+        ],
+        "installer": "docker",
+        "env": "KUBECONFORM_CONFIG"
     },
     {
         "name": "dockerfile_lint",
         "args": "-r /etc/opt/goat/oci.yml -f",
-        "filetype": "Dockerfile",
-        "installer": "npm"
+        "filetype": [
+            "Dockerfile"
+        ],
+        "installer": "npm",
+        "env": "DOCKERFILE_LINT_CONFIG"
     },
     {
         "name": "markdown-link-check",
         "args": "--config /etc/opt/goat/links.json --verbose",
-        "filetype": ".md",
+        "filetype": [
+            ".md"
+        ],
         "installer": "npm",
-        "executor": "npx"
+        "executor": "npx",
+        "env": "MARKDOWN_LINK_CHECK_CONFIG"
     },
     {
         "name": "cspell",
         "args": "-c /etc/opt/goat/cspell.config.js -v --no-must-find-files --",
-        "filetype": ".md",
+        "filetype": [
+            ".md"
+        ],
+        "installer": "npm",
+        "executor": "npx",
+        "env": "CSPELL_CONFIG"
+    },
+    {
+        "name": "cfn-lint",
+        "args": "-D --config-file /etc/opt/goat/.cfnlintrc.yml",
+        "filetype": [
+            ".yml",
+            ".yaml"
+        ],
+        "installer": "pip",
+        "env": "CFN_LINT_CONFIG"
+    },
+    {
+        "name": "jscpd",
+        "args": "--config /etc/opt/goat/.jscpd.json *",
+        "filetype": [
+            "all"
+        ],
+        "installer": "npm",
+        "executor": "npx",
+        "env": "JSCPD_CONFIG"
+    },
+    {
+        "name": "actionlint",
+        "args": "-config-file /etc/opt/goat/actionlint.yml",
+        "filetype": [
+            "all"
```

Comment:
Aren't these just `.yml` or `.yaml` file? From [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#about-yaml-syntax-for-workflows):
> Workflow files use YAML syntax, and must have either a `.yml` or `.yaml` file extension. If you're new to YAML and want to learn more, see "[Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml/)."

Reply:
Done